composer.json

```diff
@@ -45,7 +45,8 @@
     "ocramius/proxy-manager": "^2.6",
 
     "elasticsearch/elasticsearch": "^7.3",
-    "monolog/monolog": "^2.0"
+    "monolog/monolog": "^2.0",
+    "symfony/debug": "^4.4"
```

Comment:
Could you install this on the `dev` dependencies? 😊

Reply:
I've tried and still not working.
Apparently is something related to the dependencies.

Also, I am not sure but I think the `clever/prestissimo` docker image (inside the Makefile) is not installed.
Could you take a look @rgomezcasas? :) 

Thank you!!

Reply:
@rgomezcasas done 👍🏽